charts/src/main/java/hu/ma/charts/line/data/LineChartData.kt

```diff
@@ -0,0 +1,38 @@
+package hu.ma.charts.line.data
+
+import android.graphics.Typeface
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+
+data class LineChartData(
+  val series: List<SeriesData>,
+  val xLabels: List<Any> = emptyList(),
+  val xLabelFormatter: (label: Any) -> String = { label -> label.toString() },
```

Comment:
Why `List<Any>` if it ends up being mapped to `String` either way?

Reply:
In my case I am sending in labels as dates, but want them formatted in a specific way. This is just the default formatter for any other case where you don't want anything else than just the string representation.

Reply:
Yep, but unless I'm missing something it is the same thing as passing already formatted `String`s into `xLabels` 🤔

Reply:
You're absolutely right. It's a leftover from an earlier attempt at passing on data when drilling down into the chart, but I reverted that but left this behind. Good catch - thanks!